covid_microservice/config.py

```diff
@@ -17,16 +17,22 @@
 
 from ghga_service_chassis_lib.api import ApiConfigBase
 from ghga_service_chassis_lib.config import config_from_yaml
+from hexkit.providers.mongodb.provider import MongoDbConfig
 
-from .models import SupportedLanguages
+from .core.models import SupportedLanguages
 
 
-# Please adapt config prefix and remove unnecessary config bases:
-@config_from_yaml(prefix="my_microservice")
+class SamplesDaoFactoryConfig(MongoDbConfig):
+    """Contains config for DAO Factory"""
+
+    collection_name = "samples"
+
+
+@config_from_yaml(prefix="covid_microservice")
```

Comment:
Prefer something shorter here:

```suggestion
@config_from_yaml(prefix="cm")
```